### PR TITLE
Automate version bump

### PR DESCRIPTION
**Pull request by seallard**
# Summary
Automatically run the version bump when a merge is made to master.

### Add
- Bump workflow

### Change
- Update deployment instructions

## Review
- [ ] code approved by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: .github/workflows/bumpversion.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    runs-on: ubuntu-latest
+    name: Bump version and push tags to master
+    steps:
+      - name: Bump version
+        uses: Clinical-Genomics/bump2version-ci@v3
+        env:
+          BUMPVERSION_TOKEN: ${{ secrets.BUMPVERSION_TOKEN }}
+          BUMPVERSION_AUTHOR: ${{ secrets.BUMPVERSION_AUTHOR }}
+          BUMPVERSION_EMAIL: ${{ secrets.BUMPVERSION_EMAIL }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
```
Comment (henrikstranneheim) [inline .github/workflows/bumpversion.yml:17]: Add new line

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -1,35 +1,20 @@
...
+2. Append version increment value `( major | minor | patch )` in the commit message to specify what kind of release is to be created.
+3. Review the details and merge the branch into master.
+4. First deploy on stage so log into hasta and run:
     - `us`
     - `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-housekeeper-stage.sh master`
-1. Deploy in productions by running the following commands:
+5. Deploy in productions by running the following commands:
     - `down`
     - `up`
     - `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-housekeeper-prod.sh`
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:16]: I think HK should use the same standard deploy script as cg and trailblazer and other python tools

**Review by henrikstranneheim — COMMENTED** [review]
Nice 👍

File: DEPLOYMENT.md (hunk 2 above)
Comment (seallard) [reply DEPLOYMENT.md:16]: The standard deploy script throws an error
```bash
FAILED: No config file '/home/proj/stage/servers/alembic/housekeeper/alembic-stage.ini' found, or file has no '[alembic]' section
```

I'll update the deployment script to handle this. Maybe a flag could be added to indicate whether alembic is used.

**Comment by henrikstranneheim** [comment]
The other script already has a flag for that. You should be able to use it out of the box. Just supply the correct conda env and branchSkickat från min iPhone7 mars 2023 kl. 13:31 skrev Sebastian Allard ***@***.***>:﻿
@seallard commented on this pull request.



In DEPLOYMENT.md:
>      - `us`
     - `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-housekeeper-stage.sh master`
-1. Deploy in productions by running the following commands:
+5. Deploy in productions by running the following commands:
     - `down`
     - `up`
     - `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-housekeeper-prod.sh`

The standard deploy script throws an error
FAILED: No config file '/home/proj/stage/servers/alembic/housekeeper/alembic-stage.ini' found, or file has no '[alembic]' section
I'll update the deployment script to handle this. Maybe a flag could be added to indicate whether alembic is used.

**Comment by seallard** [comment]
@henrikstranneheim I just noticed it after I posted my comment haha
Updated the instructions now, works as expected.

**Review by henrikstranneheim — APPROVED** [review]
Avesome 👍

**Pull request merged**